README.md

```diff
@@ -337,6 +360,34 @@ STATICFILES_FINDERS = [
 ]
 ```
 
+### Adding support for JS and CSS
+
+If you want to use JS or CSS with components, you will need to:
+
+1. Add [`ComponentDependencyMiddleware`](#setting-up-componentdependencymiddleware) to `MIDDLEWARE` setting.
```

Comment:
The installation now tells people to ALWAYS use the middleware. And it's only if users read [the docs further](https://github.com/EmilStenstrom/django-components/pull/688/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3225), where they will learn that there are cases where they don't actually need to use the middleware.

[Also there is no more a setting that decides whether to use the middleware or not. ](https://github.com/EmilStenstrom/django-components/pull/688/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L3120)